### ISSUE TITLE
concept code for dx_origin from CDM 3.1?

### ISSUE THREAD

**Issue by dckc** [SCILHS/i2p-transform#53]
In a [May 3 GPC comment][578] Laurel V. writes:
>dx_origin - this will likely be hardcoded either in the data load ETL to avoid a local modification to i2p. Checked SCILHS to see if there was a modifier to capture this; no dice. Need to decide on a concept code; looking for input from SCILHS, if possible, to avoid rework.

[578]: https://informatics.gpcnetwork.org/trac/Project/ticket/578#comment:8

I'm interested to learn about any progress on this as it becomes available.

cc @jklann @mprittie

**Comment by jklann** [comment]
Yes, I have it in the version of CDM3.1 ontology I am working on. I am toying with pulling this mapping from sourcesystem_cd rather than modifier_cd, since this data element does seem to be asking about source system.
